### PR TITLE
Bump wmii_hg and libixp_hg

### PR DESCRIPTION
**Pull request by armeenm**
###### Description of changes
These packages have moved homes and haven't been touched in 10 years. This PR also removes Python support from wmii_hg as it depends on python2.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/window-managers/wmii-hg/default.nix [hunk 1]
```diff
@@ -29,16 +43,27 @@ stdenv.mkDerivation rec {
     EOF
   '';
 
-  nativeBuildInputs = [ pkg-config unzip ];
-  buildInputs = [ libixp_hg txt2tags dash python2 which
-                  libX11 libXrender libXext libXinerama libXrandr libXft ];
+  postInstall = ''
+    rm -rf $out/lib/python* $out/etc/wmii-hg/python
```
Comment (SuperSandro2000) [inline pkgs/applications/window-managers/wmii-hg/default.nix:47]: Please add a comment why we do this.

**Review by SuperSandro2000 — COMMENTED** [review]
Why to the packages end in `-hg`?

Please replace patchPhase with postPatch and configurePhase with postConfigure.

**Comment by armeenm** [comment]
> Why to the packages end in -hg?

I'm not sure exactly what the history is, but I think at some point wmii was (significantly(?)) improved and that coincided with a move to Mercurial. Of course, it's using Git now, so... :man_shrugging:

**Comment by SuperSandro2000** [comment]
I think we should rename it and remove -hg. If we create an alias we don't break the configs of people.

**Pull request merged**